CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for mypy-json-report
 
+## Unreleased
+
+- Add Python 3.11 to test matrix
```

Comment:
**Suggestion:** It's worth mentioning that the upper limit was removed from the python dependency too.